Annotations/.NETCore/System.Private.CoreLib/Attributes.xml

```diff
@@ -6072,6 +6072,24 @@
   <member name="M:System.Runtime.InteropServices.Marshal.PtrToStringUni(System.IntPtr)">
     <attribute ctor="M:JetBrains.Annotations.CanBeNullAttribute.#ctor"/>
   </member>
+  <member name ="M:System.Runtime.InteropServices.Marshal.PtrToStructure(System.IntPtr, System.Type)">
+    <attribute ctor="M:JetBrains.Annotations.ContractAnnotationAttribute.#ctor(System.String)">
+      <argument>ptr:null=&gt;null</argument>
+    </attribute>
+    <attribute ctor="M:JetBrains.Annotations.NotNullAttribute.#ctor" />
```

Comment:
result is null when ptr is "IntPtr.Zero"

Reply:
This is exactly what this annotation does. See comment above.